### ISSUE TITLE
Favorites System

### ISSUE THREAD

**Issue by PatheticMustan** [grgwnw/PartsApp#4]
As I said in #2, we need to have a system for favorites that save locally  to the device.
It seems like the best solution would be [AsyncStorage](https://facebook.github.io/react-native/docs/asyncstorage).

We can use this to store string key-value pairs.

Right now I'm thinking we can have a list of integer IDs for each item/tool, and just join them in a string later.
[1, 2, 3] would be 1_2_3, and we can just use String#split or String#join to parse/stringify them I guess

**Comment by KryptoCrash** [comment]
Rather than do that, keep all items in Async storage by stringifying the json object for each item. Then unstringifying it later when in use.

**Comment by PatheticMustan** [comment]
No, it'd be better to save it as an ID, because if we later change the description, the stringified JSON object would have the old description.
Using an ID, it would update with the new data from the server.

**Comment by PatheticMustan** [comment]
I need your thoughts, ideas, feedbackkkk

**Comment by KryptoCrash** [comment]
We need a way to store an array of objects

**Comment by KryptoCrash** [comment]
ids*
